homeassistant/components/mqtt/vacuum/schema_legacy.py

```diff
@@ -204,7 +238,7 @@ def _setup_from_config(self, config):
         self._send_command_topic = config.get(CONF_SEND_COMMAND_TOPIC)
 
         self._payloads = {
-            key: config.get(key)
+            key: config[key]
```

Comment:
This change smells like it could throw a KeyError..? 

Should this also be guarded with `if key in config` like the `self._templates` dictcomp further down.